### PR TITLE
relnote(106) maction semantics MathML render first child by default

### PR DESCRIPTION
**Pull request by bsmth**
Changes in MathML rendering for `<semantics>` element in Fx 106:

>By default, only the first child of the <semantics> element is rendered while the others have their CSS `display` set to `none`.

### Related issues

- [x] Parent issue: https://github.com/mdn/content/issues/20919

### CODE REVIEW

**Comment by fred-wang** [comment]
I think we need to say something similar for `<maction>`.

File: files/en-us/mozilla/firefox/releases/106/index.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ This article provides information about the changes in Firefox 106 that will aff
 
 #### Removals
 
+### MathML
+
+- The [`<semantics>` MathML](/en-US/docs/Web/MathML/Element/semantics) and [`<maction>` MathML](/en-US/docs/Web/MathML/Element/maction) elements now only render the first child element by default to align with the [MathML Core specification](https://w3c.github.io/mathml-core/#semantics-and-presentation).
+  The rendering algorithm is described in more details in the [`<semantics>`](/en-US/docs/Web/MathML/Element/semantics#sect1) page ({{bug(1588733)}}).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/106/index.md:24]:
```suggestion
- The [`<semantics>`](/en-US/docs/Web/MathML/Element/semantics) and [`<maction>`](/en-US/docs/Web/MathML/Element/maction) MathML elements now only render the first child element by default ({{bug(1588733)}}).
```
Removed the last bits - we probably don't need to include that info here. I guess the links to the two pages are enough should the readers want to see more details/spec.

**Review by dipikabh — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/firefox/releases/106/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/mozilla/firefox/releases/106/index.md:24]: OK great, let's do that

**Review by dipikabh — APPROVED** [review]
Thanks! 👍

**Pull request merged**